src/BloomExe/Book/Book.cs

```diff
@@ -2062,13 +2062,18 @@ private void WriteLanguageDisplayStyleSheet( )
 
 			using (var temp = TempFile.WithExtension(".css"))
 			{
-				RobustFile.WriteAllText(temp.Path,
+				// Use a temporary file pathname in the current book's folder.  This is needed to ensure proper permissions are granted
```

Comment:
I'm wondering if there is a natural way to capture this pattern you're doing... like

``` using(var temp = TempFile.InFolder(_storage.FolderPath).WithExtension(".css"); 

Though in this case at least, there isn't a need for the extension if I'm not mistaken, as we later copy it over.
If that is always going to be the case (and maybe it is), then just implementing TempFile.InFolder() would be enough.